### DISCUSSION TITLE
add infinit line in chart

### DISCUSSION THREAD

**Discussion by armeya25** [highfestiva/finplot#474] [General]
how do i add pg.InfiniteLine in chart,

i am trying to add following in chart

v_bar = pg.InfiniteLine(pos=(position), angle=90, pen='r')
ax.vb.addItem(v_bar)

**Comment by highfestiva** [comment]
Looks right, you probably have the wrong position. Use the 0-based index of the X-coordinate (which usually is displayed as a timestamp). For instance, in `examples/line.py`, this works:
```python
v_bar = fplt.pg.InfiniteLine(pos=(600,4.5), angle=0.01, pen='r')
ax.vb.addItem(v_bar)
```
